src/lib/parse/parseContextAttributeCompoundValue.h

```diff
@@ -0,0 +1,49 @@
+#ifndef SRC_LIB_PARSE_PARSECONTEXTATTRIBUTECOMPOUNDVALUE_H_
```

Comment:
However, we have lib/serviceRoutinesV2 and lib/serviceRoutines...Why aren't we doint the same for the parser library? I will be easier to remove v1.